frontend/app/routes/protected/person-case/primary-docs.tsx

```diff
@@ -16,10 +16,23 @@ import { FetcherErrorSummary } from '~/components/error-summary';
 import { InputSelect } from '~/components/input-select';
 import { PageTitle } from '~/components/page-title';
 import { Progress } from '~/components/progress';
+import { AppError } from '~/errors/app-error';
+import { ErrorCodes } from '~/errors/error-codes';
 import { getFixedT } from '~/i18n-config.server';
 import { handle as parentHandle } from '~/routes/protected/layout';
 import { getLanguage } from '~/utils/i18n-utils';
 
+type PrimaryDocumentsSessionData = NonNullable<SessionData['inPersonSINCase']['primaryDocuments']>;
+
+/**
+ * Valid current status in Canada for proof of concept
+ */
+const VALID_CURRENT_STATUS = ['canadian-citizen-born-outside-canada'] as const;
+/**
+ * Valid document type for proof of concept
+ */
+const VALID_DOCTYPE = ['certificate-of-canadian-citizenship', 'certificate-of-registration-of-birth-abroad'] as const;
```

Comment:
Remove `as const` since those values doesn't need to be string literals and it causes issue while build the validation schema.